CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
 ## [Unreleased 0.3.1]
 
+
+
 ### Added
 
 - Self-hosted etcd: if `etcd-hosts.checkpoint` file exists under `${datadir}/`,
   etcd pod will restore the hosts mapping from it before etcd bootstraps.
 - Add static TLS support for self-hosted etcd mode.
+- The operator will now post Kubernetes [events](https://kubernetes.io/docs/resources-reference/v1.6/#event-v1-core). To allow this the necessary RBAC rule for the resource `events` must be added to the clusterrole. See the [rbac guide](https://github.com/coreos/etcd-operator/blob/master/doc/user/rbac.md#create-clusterrole) to see how to set up RBAC rules for the operator.
```

Comment:
mention that etcd operator prints out errors after upgrade if rbac is not setup correctly. but it wont affect the operation of the operator.